Automattic/wp-calypso
# Add/stepper import flow

#### Proposed Changes

Introduced a new `import-focused` onboarding flow. Now, we can directly point users to the import flow bypassing the goals step.

#### Testing Instructions

* Go to `/setup/import?siteSlug={SLUG}&flow=import-focused`
* Test each importer flow
* **NOTE:** This chunk of changes doesn't contain any update on the importer step, so it should remain the same since it re-usage of existing import steps

#### Pre-merge Checklist






Related to #68656
Closes #68392
